### PR TITLE
Switch face recognition off

### PR DESCRIPTION
**Pull request by thulethi**
Jira: https://netguru.atlassian.net/browse/SMAR-275

- Add column `face_recognition_enabled` to User model
- Add route to toggle face_recognition_enabled for specified user
- Add change to `verify` method so that only users who have face_recognition_enabled set to True would be queried.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -36,6 +37,20 @@ def hello():
...
+@app.route('/toggle_face_recognition', methods=['POST'])
+def toggle_face_recognition():
+    user = next(iter(User.query.filter_by(email=request.args['email']).all()), None)
+    if user:
+        user.face_recognition_enabled = not user.face_recognition_enabled
+        try:
+            db.session.add(user)
+            db.session.commit()
+            return jsonify({'success switching face recognitionn to': user.face_recognition_enabled})
+        except:
```
Comment (Morbidetto) [inline main.py]: Do not use bare except, try only to catch errors which we expect to happen during try block. In this case SQLAlchemyError. I'm thinking we should also return or log the error.

**Comment by Morbidetto** [comment]
Don't we also need a method to check current value of face_recognition_enabled?

File: main.py [hunk 2]
```diff
@@ -46,6 +46,37 @@ def hello():
     return '<h1>Hello Netgural!</h1>'
 
 
+@app.route('/toggle_face_recognition', methods=['POST'])
+def toggle_face_recognition():
+
+    if not 'email' in request.args:
+        return jsonify({'error': 'No email in params'}), 400
+
+    user = next(iter(User.query.filter_by(email=request.args['email']).all()), None)
```
Comment (mvbrn) [inline main.py]: https://docs.sqlalchemy.org/en/13/orm/query.html#sqlalchemy.orm.query.Query.one

Comment (Morbidetto) [reply main.py]: But then we must handle an exception in case if the user with this email doesn't exist. Current solution will return None in this case.

Comment (mvbrn) [reply main.py]: So what is the problem with handling an exception?

Comment (Morbidetto) [reply main.py]: @mvbrn Please take a look at new commit 🙂

**Pull request merged**